flipperdevices/flipperzero-firmware
# [FL-2212] File validators and archive fixes

# What's new

- Validating filename on rename
- 125khz: trim long file names on delete screen
- Lock menu: cursor position save on entering Set PIN
- Archive: various fixes

# Verification 

- Try to rename file in all apps
- Check archive app

# Checklist (do not modify)
